custom/controllers/TeensyController.java

```diff
@@ -0,0 +1,34 @@
+package org.usfirst.frc4904.standard.custom.controllers;
+
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.Joystick;
+
+public class TeensyController extends Joystick implements Controller {
+	public CustomButton[] buttons = new CustomButton[30];
+	public final int port;
+
+	public TeensyController(int port) {
+		super(port);
+		this.port = port;
+		for (int index = 1; index < 31; index++) {
+			buttons[index] = new CustomButton(this, index);
```

Comment:
I would suggest making the buttons zero-indexed, the one-indexing on the other joysticks is suboptimal.